.github/PULL_REQUEST_TEMPLATE

```diff
@@ -86,11 +85,6 @@ Fixes: #____
 
 #### Screenshots
 <!--(if appropriate)-->
-#### Checklist:
```

Comment:
❗️ I disagree about removing this. I always check this when reviewing PRs. Does everyone else feel like they're not used?

Reply:
I personally like the documentation one and the test ones. If at the very least they make the developer consider if they forgot any associated docs/tests updates that should happen in addition to the code changes.

Reply:
I like this section. If reviewers don't use them, at the very least it forces the developer to consider if any areas mentioned in the checklist should be updated. Every item in this check list should be considered when making code changes.